src/transformer.ts

```diff
@@ -135,9 +135,9 @@ export default function transformer(
         ...tsTransformPathsContext,
         sourceFile,
         isDeclarationFile: sourceFile.isDeclarationFile,
-        originalSourceFile: (<typeof ts>tsInstance).getOriginalSourceFile(sourceFile),
+        originalSourceFile: tsInstance.getSourceFileOfNode(sourceFile),
```

Comment:
I wonder if, for backward compatibility, it would make more sense to do something like this?

```
originalSourceFile: (<typeof ts>tsInstance).getSourceFileOfNode
  ? (<typeof ts>tsInstance).getSourceFileOfNode(sourceFile)
  : (<typeof ts>tsInstance).getOriginalSourceFile(sourceFile),
```

That way, we use the new method if it exists, or fall back to the old one.

Reply:
Actually, the newly suggested method here (`getSourceFileOfNode`) is also marked as `@internal`. I have a PR that uses public methods and adds test coverage here: https://github.com/LeDDGroup/typescript-transform-paths/pull/301